src/main/resources/templates/create.html

```diff
@@ -1,10 +1,128 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
 <head>
     <meta charset="UTF-8">
-    <title>Title</title>
+    <title>Create Survey</title>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
+    <script type="text/javascript">
+        var questionCount = 3;
+
+        $(document).ready(function(){
+            $( ".addMCQuestion" ).on("click", function(){
+                var $clone = $( "ul.MCList" ).first().clone(true);
+                $clone.append( "<button type='button' class='remove-row'>-</button>" );
+                $clone.insertBefore( ".addMCQuestion" );
+                console.log("added MC question");
+                questionCount++;
+            });
+
+            $( ".addNRQuestion" ).click(function(){
+                var $clone = $( "ul.NRList" ).first().clone(true);
+                $clone.append( "<button type='button' class='remove-row'>-</button>" );
+                $clone.insertBefore( ".addNRQuestion" );
+                console.log("added NR question");
+                questionCount++;
+            });
+
+            $( ".addOEQuestion" ).click(function(){
+                var $clone = $( "ul.OEList" ).first().clone(true);
+                $clone.append( "<button type='button' class='remove-row'>-</button>" );
+                $clone.insertBefore( ".addOEQuestion" );
+                console.log("added OE question");
+                questionCount++;
+            });
+
+            $( ".addChoice" ).on("click", function(){
+                var $clone = $( "li.choice" ).first().clone(true);
+                $clone.append( "<button type='button' class='removeChoice'>-</button>" );
+                $clone.insertBefore($(this));
+                console.log("added MC choice");
+            });
+
+            $( ".createSurvey" ).on("click", ".remove-row", function(){
+                $(this).parent().remove();
+            });
+
+            $( ".multipleChoice" ).on("click", ".removeChoice", function(){
+                $(this).parent().remove();
+            });
+
+            $('.submit').on("click", function() {
+                var formData = JSON.stringify($(".createSurvey").serializeArray());
+                console.log(formData);
+                $.ajax({
+                    type: 'POST',
+                    url: "createSurvey",
+                    dataType: 'json',
+                    contentType: "application/json; charset=utf-8",
+                    data: formData
+                });
+                var baseURL = window.location.origin;
+                window.location.href = baseURL;
```

Comment:
Change this part to this:
`var baseURL = window.location.origin;

            $('.submit').on("click", function() {
                var formData = JSON.stringify($(".createSurvey").serializeArray());
                console.log(formData);
                $.ajax({
                    type: 'POST',
                    url: baseURL+"/createSurvey",
                    dataType: 'json',
                    contentType: "application/json; charset=utf-8",
                    data: formData
                });

                setTimeout(function(){
                    window.location.href = baseURL;
                }, 15);`

Sometimes the submission ajax request does not go through because the redirection is too fast